js/src/test/scala/com/raquo/waypoint/BasePathSpec.scala

```diff
@@ -33,41 +33,41 @@ class BasePathSpec extends UnitSpec {
 
       describe(s"basePath = `$basePath`") {
 
-        val homeRoute: Route[HomePage.type, Unit] = Route.static(
+        val homeRoute: Route.Total[HomePage.type, Unit] = Route.staticTotal(
```

Comment:
You've replaced `static` with `staticTotal` here, but that leaves `static` untested. Since `static` is the primary method, it should retain its tests, and `staticTotal` tests should be added either in this file or separately.